contrib/babelfishpg_tsql/src/backend_parser/parser.c

```diff
@@ -180,6 +180,9 @@ pgtsql_base_yylex(YYSTYPE *lvalp, YYLTYPE * llocp, core_yyscan_t yyscanner)
 		case OUTER_P:
 			cur_token_length = 5;
 			break;
+		case ALTER:
```

Comment:
What's difference between 
TSQL_ALTER and ALTER

Reply:
```
PG_KEYWORD("alter", ALTER, UNRESERVED_KEYWORD)
```

My understanding of TSQL_ALTER is that it's the token that represents Alter Procedure. We check the parser to see if we hit "alter", then check the next token to see if its "proc" or "procedure", in which case we set the current token to TSQL_ALTER

Reply:
These changes were made in order to avoid parser conflicts due to native PG's interpretation of ALTER PROCEDURE